pkg/api/pod/util_test.go

```diff
@@ -1187,24 +1175,9 @@ func TestDropPodSysctls(t *testing.T) {
 			hasSysctls:  true,
 			pod:         podWithSysctls,
 		},
-		{
-			description: "does not have Sysctls",
-			hasSysctls:  false,
-			pod:         podWithoutSysctls,
-		},
-		{
-			description: "does not have SecurityContext",
-			hasSysctls:  false,
-			pod:         podWithoutSecurityContext,
-		},
-		{
-			description: "is nil",
-			hasSysctls:  false,
-			pod:         func() *api.Pod { return nil },
-		},
 	}
 
-	for _, enabled := range []bool{true, false} {
+	for _, enabled := range []bool{true} {
```

Comment:
This can be simplified out I think, we can just set enabled = true and remove the loop.